### ISSUE TITLE
training memory fluctuates

### ISSUE THREAD

**Issue by oujieww** [pytorch/pytorch#9815]
is there any one have this kand of problem,but while training go on, the memory is changing all the time,from 2G to 11G, anyone can help me ?XD thank you
i have tryed python2.7 with pytorch0.2  and python3.5 with pytorch0.4 
after epoch 1 ,the memory is change small and loss change small ,This is so ~~~ i have no idea about this....

i have check code with my other project, the only one different area is this one use scheduler by ReduceLROnPlateau 

scheduler.step(val_loss)

**Comment by soumith** [comment]
please use https://discuss.pytorch.org for questions. Also provide a clear reproducible to help us help you.
